lib/diagnostic.js

```diff
@@ -17,7 +17,13 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+let event = (host, name, date, time); {
```

Comment:
This just needed to follow the same structure as the examples given above - the goal was to abstract properties of the calendar app.